fetch_worker.go

```diff
@@ -135,12 +136,17 @@ func (w *fetchWorker) selectRowsChunk(
 	ctx context.Context, tx *sqlx.Tx,
 	start, end int64,
 ) (scannedRows []opencdc.StructuredData, err error) {
-	query := fmt.Sprint(`
-		SELECT *
-		FROM `, w.config.table, `
-		WHERE `, w.config.primaryKey, ` > ? AND `, w.config.primaryKey, ` <= ?
-		ORDER BY `, w.config.primaryKey, ` LIMIT ?
-	`)
+	query, args, err := squirrel.Select("*").
+		From(string(w.config.table)).
+		Where(squirrel.Gt{(string(w.config.primaryKey)): start}).
+		Where(squirrel.LtOrEq{string(w.config.primaryKey): end}).
```

Comment:
I originally added the types so that it makes it more difficult to pass the wrong string argument into a function. You'd get a type error if confusing the two.

Then again, this is already covered in the integration tests, and there's not really that much code that makes this useful. And it is uncomfortable to be casting the types everywhere I use `PrimaryKeyName`, for example. In tests I also get errors if I try to compare a `PrimaryKeyName` to a string, for example.

So yeah, I want to keep this simple, I now don't really think that it is as useful as I once thought. Specially when I already got a few integration tests covering the errors that this should in theory prevent.

Reply:
@hariso just removed all primitive nominal types from the project. There was a bit of benefit of self documenting types at `common/position.go`, but added a few comments to clarify what is what, so all should be fine and simpler.

lmk what you think and whether we can get this merged.

Reply:
> I originally added the types so that it makes it more difficult to pass the wrong string argument into a function. You'd get a type error if confusing the two.

Agreed, it's definitely a nice benefit. However, I think that in our case, the risk of mixing those up is pretty low. Otherwise, anytime there are two strings/integers, etc. a new type would be needed.